### DISCUSSION TITLE
Spec file stuck/freeze on loading

### DISCUSSION THREAD

**Discussion by abuzarzia** [cypress-io/cypress#28590] [Questions and Help]
After the new update 13.6.1 the cypress spec froze on loading. 
1: First-time spec is running fine
2: After making some changes in the file When I press (ctrl + s)  the file is stuck on load.

But when I downgrade the cypress version from 13.6.1 to 13.5.0 it's working fine.

Can anyone explain what is the reason behind this?
I'm using, Windows 10 Pro, version 22H2

![Load Freeze](https://github.com/cypress-io/cypress/assets/72908295/2adbff7a-b349-43eb-9b3b-4dda5f67cf3d)

**Comment by jennifer-shehane** [comment]
Looks like this issue https://github.com/cypress-io/cypress/issues/28392
